app/javascript/lib/vue-components/modules/BeesWarmChart.vue

```diff
@@ -207,18 +184,43 @@ export default {
         )
         .call(axisY);
 
+      g
+        .selectAll('.label-y')
+        .data(Array.from(new Set(filterData.map(d => d[this.yAxisProp]))))
+        .join('text')
+        .attr('class', 'label-y')
+        .attr('x', 0)
+        .attr('y', d => scaleY(d))
+        .attr('alignment-baseline', 'middle')
+        .text(d => d);
+
+      g
+        .selectAll('.line-y')
+        .data(Array.from(new Set(filterData.map(d => d[this.yAxisProp]))))
+        .join('line')
+        .attr('class', 'line-y')
+        .attr('x1', this.margin.left)
+        .attr('x2', this.svgWidth + this.margin.left + this.margin.right)
+        .attr('y1', d => scaleY(d))
+        .attr('y2', d => scaleY(d));
+
+      g.selectAll('.label-y').call(this.wrapTextLabel, 120);
+
       let simulation = d3
         .forceSimulation(filterData)
         .force('x', d3.forceX(d => scaleX(d[this.xAxisProp])).strength(2))
         .force('y', d3.forceY(d => scaleY(d[this.yAxisProp])))
         .force('collide', d3.forceCollide().radius(d => d.radius + this.padding))
         .stop();
 
-      for (let i = 0; i < (filterData.length / 3); ++i) {
-        simulation.tick(5);
+      /*Reduces the simulation to prevent brokes the browser*/
+      let simulationIteration = 85
```

Comment:
What's that?

Reply:
Force Layout needs `tick()` to run the simulation and place elements in her correct position. First, I used the data's length, but a lot of nodes/circles run simulation causes the browser to be frozen. After a few tests run 85 simulations, work well.

[d3-force](https://github.com/d3/d3-force#simulation_tick)